tests/test_timing_properties.py

```diff
@@ -0,0 +1,143 @@
+"Contains a collection of pytest tests that validates the timing properties."
+
+import random
+
+import pytest
+
+import nidaqmx
+import nidaqmx.system
+from nidaqmx import DaqError
+from nidaqmx.constants import AcquisitionType, SampleTimingType
+
+
+class TestTimingProperty(object):
+    """Contains a collection of pytest tests that validates the timing properties."""
+
+    def test_boolean_property(self, any_x_series_device):
+        """Test for validating boolean property."""
+        with nidaqmx.Task() as task:
+            task.ao_channels.add_ao_voltage_chan(any_x_series_device.ao_physical_chans[0].name)
+
+            task.timing.samp_timing_type = SampleTimingType.ON_DEMAND
+
+            # Test property initial value.
+            assert not task.timing.simultaneous_ao_enable
+
+            # Test property setter and getter.
+            task.timing.simultaneous_ao_enable = True
+            assert task.timing.simultaneous_ao_enable
+
+            # Test property deleter.
+            del task.timing.simultaneous_ao_enable
+            assert not task.timing.simultaneous_ao_enable
+
+    def test_string_property(self, any_x_series_device):
+        """Test for validating string property."""
+        with nidaqmx.Task() as task:
+            task.ai_channels.add_ai_voltage_chan(any_x_series_device.ai_physical_chans[0].name)
+
+            task.timing.cfg_samp_clk_timing(1000)
+
+            default_value = task.timing.samp_clk_src
+
+            # Test property default value.
+            assert task.timing.samp_clk_src == default_value
+
+            # Test property setter and getter.
+            task.timing.samp_clk_src = "PFI0"
+            assert task.timing.samp_clk_src == "/nidaqmxMultithreadingTester1/PFI0"
```

Comment:
`"/nidaqmxMultithreadingTester1/PFI0"` should be `f"/{any_x_series_device.name}/PFI0"`

Reply:
Thank you for the solution.